build/platforms.bzl

```diff
@@ -36,6 +36,23 @@ UBI_PLATFORMS = {
     ],
 }
 
+CLIENT_PLATFORMS = {
+    "linux": [
+        "amd64",
+        "arm64",
+        "arm",
+        "ppc64le",
+        "s390x",
+    ],
+    "darwin": [
+        "amd64",
+        "arm64",
```

Comment:
You are _really_ keen to get kubectl running on your iPad aren't you? 😄

Reply:
Go fails on it :(